app/experimenter/experiments/views.py

```diff
@@ -230,3 +230,19 @@ def form_invalid(self, form):
 
 class NimbusUIView(TemplateView):
     template_name = "nimbus/index.html"
+
+
+class PageNotFoundView(TemplateView):
+    template_name = "nimbus/404.html"
+
+    @classmethod
+    def as_404_view(cls):
+        as_view_fn = cls.as_view()
+
+        def view_fn(request, exception):
+            response = as_view_fn(request)
+            response.status_code = 404
+            response.render()
+            return response
+
+        return view_fn
```

Comment:
Without L238 - L248 + using `as_view` rather than `as_404_view`, the response code is still a `200` despite using Django's `handler404`.